doc/man/scons.xml

```diff
@@ -893,40 +906,6 @@ should take place in parallel.)
   </listitem>
   </varlistentry>
 
-<varlistentry>
-  <term>--debug=action_timestamps</term>
-  <listitem>
-    <para>Prints additional time profiling information:</para>
```

Comment:
Why remove this info?

Reply:
not removed (although trimmed) - alphabetized.